wpeverest/everest-forms
# Fix - Multi-part form conflict in frontend single page

### All Submissions:






### Changes proposed in this Pull Request:



This PR support the latest changes and backport if the previous version is found for Multi-part addon.

### How to test the changes in this Pull Request:

1. Check out this PR branch.
2. In PRO checkout develop and test, no PHP notices should be detected.
2. In PRO checkout multi-part related branch and test, no PHP notices should be detected.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Multi-part form conflict in frontend single page.
